platform/api/_partials/resources/runners/reference.mdx

```diff
@@ -1075,7 +1075,7 @@ for each node that meets all of the scheduling requirements (resource
 request, requiredDuringScheduling affinity expressions, etc.),
 compute a sum by iterating through the elements of this field and adding
 "weight" to the sum if the node matches the corresponding matchExpressions; the
-nodes with the highest sum are the most preferred.
+node(s) with the highest sum are the most preferred.
```

Comment:
node should be used instead of node(s). Could you please change here and in all other occurrences of `noun(s)`

Reply:
I can, but it will be overwritten by every PR created by CI. 
This description is generated from Runner's `.spec.clusterRef.podTemplate.spec.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution` here: https://github.com/loft-sh/loft-enterprise/blob/main/staging/src/github.com/loft-sh/api/v4/pkg/apis/storage/v1/runner_types.go#L154 which is `corev1` type, so we don't have a control over godoc there.
I can fix it for now, but next time we do platform release we will need to fix it again (this applies to every change you requested in this file)

Reply:
fixed for now